### ISSUE TITLE
Provide Military Tools GP as Conda package

### ISSUE THREAD

**Issue by lfunkhouser** [Esri/military-tools-geoprocessing-toolbox#305]
Tasks to be done 16 May 2018
- [x] Add the tarbz file to the build
- [x] doc update for conda workflow
- [x] add tasks that need to be done with Matt
- [x] add number features tool
- [x] update the conda package with the latest updates to tools
- [x] Contact Paul L to create a new conda package and put on solutionsdev channel
- [x] Contact Shaun W for militarytools to be added to the Esri channel from solutionsdev channel @lfunkhouser 

- [x] Test connected on 2.2

- [x] Test disconnected on 2.2

Both Paul and Shaun are waiting for us to contact them.
-----------------------------------------------------
Providing Military Tools GP as a conda package will enable the Military Tools toolbox to be added as a system toolbox.

Since it is required that at 2.1, you need to be admin to install the conda package, both the conda package and the pyt will be included with Military Tools.

There is a limitation for ArcMap. ArcGIS Pro includes a package manager that allows you to interact directly with conda, without having to leave ArcGIS Pro.

**Comment by lfunkhouser** [comment]
Updated the comments with the tasks that are remaining. This task remains impeded until the updates to Number Features #341 , Find Local Peaks #345 and GRG from Area #343 are complete and included in the Military Tools.pyt.

**Comment by dfoll** [comment]
Doc is ready for review http://solutionsdev.arcgis.com/defense/help/military-tools/get-started/military-tools-for-arcgis-get-started.htm

**Comment by dfoll** [comment]
@lfunkhouser can we get some clarification on what "testing" means for this? Testing the entire toolbox a second time? or just testing that I can get the conda package to install the tools?

**Comment by csmoore** [comment]
For testing (disconnected) - just making sure this is the version/file to grab is from here: https://anaconda.org/solutionsdev/militarytools  (it shows built 16 days ago)

**Comment by dfoll** [comment]
#### Disconnected
- [x] 2.1 - does not work, cannot create a new Python environment
- [ ] 2.1 (run Pro as administrator)
- [x] 2.2 worked (didn't test tools)

#### Disconnected (spot checking tool functionality)
- [ ] 2.1 - does not work, cannot create a new Python environment
- [ ] 2.1 (run Pro as administrator)
- [ ] 2.2 worked

**Comment by topowright** [comment]
@lfunkhouser @dfoll I have updated the changes that we made into Paul's branch. I have spoke to Paul this morning and we should be moving forward sometime after 11am with this process. I have done all the work that I understand how to do up to this point, which was adding the script updates that we made in the last six days. Those updates are : 
Addition of the Number Features Tool
Updates to GRG from Area
Fixes made to Find Local Peaks
